NEWS.txt

```diff
@@ -19,6 +19,9 @@ Version 7.24 - not yet released
   - new asw27 polar from idaflieg
 * Windows
   - new default fonts
+  - use 6 devices in serial port list (like all other systems)
+  - allow connection to COM port 10 to 255 (f.e. for BlueTooth user)
+  - remove index based COM port setting (obsolet since XCSoar 5)
```

Comment:
typo - and even if it's obsolete, what's the justification for removing XCSoar 5 compatibility?

Reply:
To cleanup from old unused settings and debloating the system - or do you think, this is used somewhere?

Reply:
I don't know if it's used and it may (or may not) be reasonable to remove it - but you are removing this as an unrelated part of a cluttered bigger commit, and that's not reasonable.

Reply:
The commit has to be split in two parts, you are right!